staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5488,6 +5500,15 @@ type PodLogOptions struct {
 	// the actual log data coming from the real kubelet).
 	// +optional
 	InsecureSkipTLSVerifyBackend bool `json:"insecureSkipTLSVerifyBackend,omitempty" protobuf:"varint,9,opt,name=insecureSkipTLSVerifyBackend"`
+
+	// Return the specific log stream. Defaults to "all".
+	// Acceptable values are:
+	// - "all": the combined stdout and stderr
+	// - "stdout": only the stdout
+	// - "stderr": only the stderr
+	// This is an alpha field and requires enabling SplitStdoutAndStderr feature gate.
```

Comment:
Just to clarify here (to make sure I understand the intent in the KEP) - the API *will not* accept the empty string (which is our convention for new fields), but kubectl will accept `--stream ""` as the empty string?

Reply:
Hi Clayton,
I'm new to this community and i was looking for some guidance through the process and I'll be really grateful if you take some time out to help me through and become an integral part of this community.

Reply:
> but kubectl will accept --stream "" as the empty string?

Nope, the default value of flag `--stream` would be `all`, and I plan to reject the empty string.

Reply:
Ok, so both CLI and API will reject empty string?